### PR TITLE
DPM-82 Cars service example update

### PR DESCRIPTION
**Pull request by bashmach**
## Changes
- update to node 16
- switch "db" from custom session solution to sqlite3 (memory)
- update layout and copy

## Testing

Run [stackblitz](https://stackblitz.com/edit/remix-cars-service) or vercel staging ([list](https://remix-cars-service-26yd3957u-bshmch.vercel.app/list), [details](https://remix-cars-service-26yd3957u-bshmch.vercel.app/details), [settings](https://remix-cars-service-26yd3957u-bshmch.vercel.app/settings))

details
list
settings

### CODE REVIEW

File: apps/remix-cars-service/package.json [hunk 1]
```diff
@@ -11,21 +11,23 @@
   },
   "dependencies": {
     "@pipedrive/custom-app-surfaces-sdk": "^0.4.0",
```
Comment (AndreasSaltsberg) [inline apps/remix-cars-service/package.json:13]: Can you refactor this library to app-extensions-sdk?

File: apps/remix-cars-service/app/components/details/item.jsx [hunk 2]
```diff
@@ -16,47 +17,41 @@ export default function Item({ item, setItem }) {
...
-			<div className='row'>
-				<div className="label">
-					<CalendarIcon/>
-				</div>
-
-				<div className="main">
-					<div className="label">
-						<span className="font-size--s">Delivery: {item.price}</span>
+					<div className="label font-size--s">
+						<span>Condition:&nbsp;</span>
```
Comment (AndreasSaltsberg) [inline apps/remix-cars-service/app/components/details/item.jsx:30]: Probably `&nbsp;` can be replaced with whitespace?

```suggestion
						<span>Condition:{' '}</span>
```

File: apps/remix-cars-service/app/components/details/item.jsx [hunk 3]
```diff
@@ -16,47 +17,41 @@ export default function Item({ item, setItem }) {
...
+						<span className="font-size--s"><strong>New</strong></span>
 					</div>
-				</div>
-			</div>
+					<div className="label font-size--s">
+						<span>Price:&nbsp;</span>
```
Comment (AndreasSaltsberg) [inline apps/remix-cars-service/app/components/details/item.jsx:34]:
```suggestion
						<span>Price:{' '}</span>
```

File: apps/remix-cars-service/app/components/details/item.jsx [hunk 4]
```diff
@@ -16,47 +17,41 @@ export default function Item({ item, setItem }) {
...
+						<span><strong>{item.price}</strong></span>
+					</div>
+					<div className="label font-size--s">
+						<span>Status:&nbsp;</span>
```
Comment (AndreasSaltsberg) [inline apps/remix-cars-service/app/components/details/item.jsx:38]:
```suggestion
						<span>Status:{' '}</span>
```

File: apps/remix-cars-service/app/components/details/item.jsx [hunk 5]
```diff
@@ -16,47 +17,41 @@ export default function Item({ item, setItem }) {
...
+					<div className="label font-size--s">
+						<span>Delivery date: {item.delivery}&nbsp;</span>
 
-				<div className="main">
-					<div className="label">
-						<span className="person">{item.person}</span>
+						<CalendarIcon/>
 					</div>
+					<div className="label font-size--s">
+						<span>Customer name:&nbsp;</span>
```
Comment (AndreasSaltsberg) [inline apps/remix-cars-service/app/components/details/item.jsx:48]:
```suggestion
						<span>Customer name:{' '}</span>
```

**Review by AndreasSaltsberg — COMMENTED** [review]
Looks good, some small things:

**Comment by pipedrive-bot-eventsink** [comment]
### Main branch dependencies
[snyk vulnerabilities](https://app.snyk.io/org/gravity-byw/project/86786156-8dee-480d-a934-faf05b385b21)

**Review by AndreasSaltsberg — APPROVED** [review]

**Pull request merged**